config/dev.exs

```diff
@@ -75,6 +75,26 @@ config :archethic,
             Archethic.Crypto.NodeKeystore.Origin.TPMImpl
         end)
 
+config :archethic, Archethic.Utils.HydratingCache.CachesManager,
+  uco_service: [
+    {Archethic.OracleChain.Services.UCOPrice.Providers.CoinPaprika,
+     Archethic.OracleChain.Services.UCOPrice.Providers.CoinPaprika, :fetch, [["usd", "eur"]],
+     3000},
+    {Archethic.OracleChain.Services.UCOPrice.Providers.Coingecko,
+     Archethic.OracleChain.Services.UCOPrice.Providers.Coingecko, :fetch, [["usd", "eur"]], 3000},
+    {Archethic.OracleChain.Services.UCOPrice.Providers.CoinMarketCap,
+     Archethic.OracleChain.Services.UCOPrice.Providers.CoinMarketCap, :fetch, [["usd", "eur"]],
+     3000}
+  ],
+  coco_service: [
+    {:coin_paprika, Archethic.OracleChain.Services.UCOPrice.Providers.CoinPaprika, :fetch,
+     [["usd", "eur"]], 3000},
+    {:coin_gecko, Archethic.OracleChain.Services.UCOPrice.Providers.Coingecko, :fetch,
+     [["usd", "eur"]], 3000},
+    {:coin_market_cap, Archethic.OracleChain.Services.UCOPrice.Providers.CoinMarketCap, :fetch,
+     [["usd", "eur"]], 3000}
+  ]
```

Comment:
coco_service ?

Reply:
I will remove it, this was for testing multiple services

Reply:
thanks